experimenter/experimenter/nimbus_ui_new/static/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+    entry: {
+        app: './js/index.js',
+        list: './js/scripts/list.js',
```

Comment:
Can we give this a more descriptive name? Like experiments_dashboard_list  ?

Reply:
I mean I could but that's a really long name?

Reply:
I mean it doesn't have to be *that*, but something more than "list". list of what?

Reply:
Sure let's do `experiment_list` 👍